content/20.treatments.md

```diff
@@ -186,7 +186,81 @@ Summarize the state of the antiviral approach.
 
 ### Broad-Spectrum Pharmaceuticals
 
-#### Hydroxychloroquine
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
```

Comment:
'but HCQ has a better safety profile' change to 'but HCQ is regarded to be safer. '

Reply:
@nafisajadavji I just realized I can use my admin powers to convert your comments to suggestions, which should make it easier for @hufengling to incorporate them. [Here's a tutorial on how to do it](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review)

Reply:
Great, thanks!